### DISCUSSION TITLE
Compare repositories

### DISCUSSION THREAD

**Discussion by adslbarxatov** [community/community#47957] [Repositories] [answered]
### Select Topic Area

Question

### Body

Hi everyone!

We've created a fork from an existing repository. We've modified its code for our specific needs (with some fundamental changes). So, now we can't commit these modifications to the mother project (it will become non-universal if we do) or pull updates from it (this will ruin our changes).

So, we want to just compare the current state of the mother repository to the ours one. We can download its code and pass it through the GitHub Desktop using temporary directory. But this way looks like a try to get from Washington to New York through Jupiter.

So, here's the question: how to do this properly?

Best regards,
Nicolay B., FDL

**Comment by jge162** [comment, marked as answer]
Hello @adslbarxatov One way to compare the current state of the mother repository with your fork is to create a new branch in your fork and make a pull request from that branch to the mother repository. This will allow you to compare the differences between the two repositories and identify any conflicts or changes that need to be addressed. This is the best way I can think of, if you don't agree just trying to help. Bests...

**Reply by adslbarxatov** [reply]
Hi!

This looks convenient and simple. Thank you!

Best regards,
Nicloay B.
